config/rbac/operator.kyma-project.io_moduleconfig_editor_role.yaml

```diff
@@ -0,0 +1,31 @@
+# permissions for end users to edit moduleconfigs.
```

Comment:
Have these been created manually? I think the typical workflow is to use the kubebuilder [RBAC annotations](https://book.kubebuilder.io/reference/markers/rbac) in the controller which will then generate the needed ClusterRoles automatically. If created manually, I would propose to leave those out and only introduce them with mentioned approach once we work on the reconcile functionality.

E.g., in KymaController we define the needed roles [here](https://github.com/kyma-project/lifecycle-manager/blob/main/internal/controller/kyma_controller.go#L83-L97), which will then lead to the `kyma_xxx_role.yaml` files being generated.

Reply:
By now we have the sibling PR as a reference: https://github.com/kyma-project/lifecycle-manager/pull/1413/files
There, an empty controller was generated as well including the RBAC annotations. We may do the same thing here.